samples/apps/copilot-chat-app/webapi/appsettings.json

```diff
@@ -223,7 +223,5 @@
   //
   // Application Insights configuration
   //
-  "ApplicationInsights": {
-    "ConnectionString": ""
-  }
+  "APPLICATIONINSIGHTS_CONNECTION_STRING": ""
```

Comment:
Swapped to the standardized connection string string name, this will mean it "just works" as an app service.

Reply:
Yes, that makes sense.